STREAMING/RequestModel.md

```diff
@@ -11,7 +11,22 @@ void REQUEST_MODEL(Hash model);
 ```
 Request a model to be loaded into memory.
 ```
+**Note**: the function **also** accepts a model passed by ped name. (string)
+
+e.g.: RequestModel('ig_abigail')
+
+Same as: RequestModel(0x400AEC41)
+
 
 ## Parameters
 * **model**: 
 
+## Examples
+```lua
+local enemyInfo = {model ="csb_ramp_marine",
```

Comment:
A bit of a useless example as this native is usually not used in isolation (e.g. a model would usually get awaited later on).

Reply:
That code is also not very useful; you probably meant `while not HasModelLoaded(enemyInfo.model) do`. Anyhow, this change doesn't add too much extra information. The main docs repo would be a better place to document Lua/JS specific support.